### DISCUSSION TITLE
I have troubling with images display, they didn't show, it show empty list

### DISCUSSION THREAD

**Discussion by Rock8090** [datasciencecompleteguide/Machine-Learning#44] [Q&A]
import glob

import cv2 as cv
path = glob.glob('C:/Users/DELL/Desktop/Documents/Ritesh Sir/Pulses_Images/*.jpg')

cv_img = []
for img in path:
    n = cv.imread(img)
    cv_img.append(n)
    cv.imshow(cv_img)
    cv2.waitKey(0)
    cv2.destroyAllWindows() 
cv_img

**Comment by datasciencecompleteguide** [comment]
1. ` for img in glob.glob(r'C:/Users/DELL/Desktop/Documents/Ritesh Sir/Pulses_Images/*.jpg'):`
2. replace cv2 with cv
3. Check the extension of the images stored in the Pulses_images folder. (Are they jpeg or jpg or some diff. extension)

**Reply by Rock8090** [reply]
import matplotlib.pyplot as plt
import matplotlib.image as mpimg
import glob

import cv2 as cv
path = glob.glob('C:/Users/DELL/Desktop/Documents/Ritesh Sir/Pulses_Images/*.jpg')

cv_img = []
for img in path:
    n = cv.imread(img)
    cv_img.append(n)
    cv.imshow(cv_img)
    cv.waitKey(0)
    cv.destroyAllWindows() 


still i'm unble to visualize the images
